Tiltfile

```diff
@@ -206,6 +208,18 @@ def build_node_yaml():
                     "http://solana-devnet:8899",
                 ]
 
+            if pythnet:
+                container["command"] += [
+                    "--pythnetRPC",
+#                    "http://solana-devnet:8899",
+                     "http://pythnet.rpcpool.com",
+                    "--pythnetWS",
+#                   "ws://solana-devnet:8900",
+                    "wss://pythnet.rpcpool.com",
+                    "--pythnetContract",
```

Comment:
I'll switch these back to the local tilt once the first round of feedback is done